VimeoNetworking/Sources/KeychainStore.swift

```diff
@@ -81,20 +81,20 @@ final class KeychainStore
      
      - returns: data, if found
      */
-    func dataForKey(key: String) throws -> NSData?
+    func dataForKey(key: String) throws -> Data?
     {
-        var query = self.queryForKey(key)
+        var query = self.queryForKey(key: key)
         
-        query[kSecMatchLimit as String] = kSecMatchLimitOne as String
+        query[kSecMatchLimit as String] = kSecMatchLimitOne as NSString
```

Comment:
We can probably safely lose the cast to `NSString`, but `kSecMatchLimit` and `kSecReturnData` are CFStrings and the compiler complains if we don't cast.